index.html

```diff
@@ -1,15 +1,15 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset='utf-8'>
+    <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="chrome=1">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
-    <link href='https://fonts.googleapis.com/css?family=Architects+Daughter' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
-    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
-    <link rel="stylesheet" type="text/css" href="stylesheets/print.css" media="print">
-    <link rel="stylesheet" type="text/css" href="stylesheets/bootstrap.min.css" media="screen">
-    <link rel="stylesheet" type="text/css" href="stylesheets/main.css" media="screen">
+    <link href="//fonts.googleapis.com/css?family=Architects+Daughter" rel="stylesheet">
+    <link rel="stylesheet" href="stylesheets/stylesheet.css" media="screen">
+    <link rel="stylesheet" href="stylesheets/github-light.css" media="screen">
+    <link rel="stylesheet" href="stylesheets/print.css" media="print">
+    <link rel="stylesheet" href="stylesheets/bootstrap.min.css" media="screen">
+    <link rel="stylesheet" href="stylesheets/main.css" media="screen">
```

Comment:
```
index.html
|   8| ••••<link•rel="stylesheet"•type="text/css"•href="stylesheets/stylesheet.css"•media="screen">
|    | [MAJOR] HTMLLintBear:
|    | type Attributes: The default type for link tags is "text/css" so it can be safely omitted: Remove the type attribute from the link tag.

index.html
|   9| ••••<link•rel="stylesheet"•type="text/css"•href="stylesheets/github-light.css"•media="screen">
|    | [MAJOR] HTMLLintBear:
|    | type Attributes: The default type for link tags is "text/css" so it can be safely omitted: Remove the type attribute from the link tag.

index.html
|  10| ••••<link•rel="stylesheet"•type="text/css"•href="stylesheets/print.css"•media="print">
|    | [MAJOR] HTMLLintBear:
|    | type Attributes: The default type for link tags is "text/css" so it can be safely omitted: Remove the type attribute from the link tag.

index.html
|  11| ••••<link•rel="stylesheet"•type="text/css"•href="stylesheets/bootstrap.min.css"•media="screen">
|    | [MAJOR] HTMLLintBear:
|    | type Attributes: The default type for link tags is "text/css" so it can be safely omitted: Remove the type attribute from the link tag.

index.html
|  12| ••••<link•rel="stylesheet"•type="text/css"•href="stylesheets/main.css"•media="screen">
|    | [MAJOR] HTMLLintBear:
|    | type Attributes: The default type for link tags is "text/css" so it can be safely omitted: Remove the type attribute from the link tag.
```